packages/components/src/DatePicker/DatePickerActivator.test.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import { DatePickerActivator } from "./DatePickerActivator";
+import { Button } from "../Button";
+
+it("renders a button by default", () => {
+  const { container } = render(<DatePickerActivator />);
+  expect(container.querySelector("Button")).toBeInTheDocument();
+});
+
+it("renders the activator with fullWidth if it's a valid element", () => {
+  const { getByText, container } = render(
+    <DatePickerActivator
+      activator={<Button fullWidth={true} label="activate me" />}
+      fullWidth={true}
+    />,
+  );
+  expect(getByText("activate me")).toBeInTheDocument();
+  expect(
+    container.querySelector("Button").className.includes("fullWidth"),
+  ).toBeTruthy();
```

Comment:
`.toHaveClass("fullWidth")` should do it the jest way

also, use `getBy` instead of query selector

Reply:
https://github.com/GetJobber/atlantis/pull/783/commits/42f4d9a8bbd0e407ec9979c9ecd64907647ed958